### ISSUE TITLE
Image preview for markdown stopped working in 1.56.0

### ISSUE THREAD

**Issue by martinzugec-ctx** [microsoft/vscode#124027]
Issue Type: <b>Bug</b>

Image preview stopped working in new version for markdown files. Reported by my colleagues (different teams, different repositories) and I have been able to reproduce in my environment.

After downgrading to previous version, images are working as expected again.

Environment:
Version: 1.56.2 (system setup)
Commit: 054a9295330880ed74ceaedda236253b4f39a335
Date: 2021-05-12T17:13:13.157Z
Electron: 12.0.4
Chrome: 89.0.4389.114
Node.js: 14.16.0
V8: 8.9.255.24-electron.0
OS: Windows_NT x64 10.0.19043

VS Code version: Code 1.56.2 (054a9295330880ed74ceaedda236253b4f39a335, 2021-05-12T17:13:13.157Z)
OS version: Windows_NT x64 10.0.19043

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-1065G7 CPU @ 1.30GHz (8 x 1498)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.78GB (16.41GB free)|
|Process Argv|--crash-reporter-id 9a97106c-071d-4227-bba6-645f4bab3ca6|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (4)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-sidebar|Acr|1.2.0
vscode-markdownlint|Dav|0.40.4
powershell|ms-|2021.2.2
markdown-all-in-one|yzh|3.4.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
pythonvspyt700cf:30270857
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30300192
pythontb:30283811
pythonvspyt551:30291414
vspre833cf:30267465
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vscoreces:30290705
pythondataviewer:30285071
vscus158:30286553
vscgsv2ct:30301613
vscorehov:30302759
bridgeflight:30302069
vscod805cf:30301675

```

</details>

**Comment by rushilShetty** [comment]
I have the same issue. This happens to me when I try to preview any markdown files and also when I open up extensions. Please see the attached screenshots.
![image](https://user-images.githubusercontent.com/7541109/118486020-6ca8cf80-b719-11eb-9d0e-4fb87b817a0a.png)

![image](https://user-images.githubusercontent.com/7541109/118486145-8ea25200-b719-11eb-813e-e6cc41ebb1bb.png)

**Comment by martinzugec-ctx** [comment]
@rushilShetty That seems to be different problem, but I've seen others reporting it (Error loading webview).

In our case, images are simply not rendered, but rest of preview works as expected: 
![image](https://user-images.githubusercontent.com/53792597/118486453-a3fe8880-b6e7-11eb-9359-e5c61db99b0f.png)

**Comment by rushilShetty** [comment]
@martinzugec-ctx Ah ok, sorry about that, I will try to post it in the other issue. Thanks.

**Comment by martinzugec-ctx** [comment]
@rushilShetty I think this is problem you are reporting: https://github.com/microsoft/vscode/issues/122951

**Comment by rushilShetty** [comment]
@martinzugec-ctx you are right. Thank you so much :)

**Comment by martinzugec-ctx** [comment]
@mjbvz Hello Matt,

markdown all in one was disabled, the issue still remains. As I've mentioned before, I have more colleagues reporting the same problem and their VSC configuration is different.

Can you please re-open this ticket, as it hasn't been caused by an extention? Have you been able to reproduce this problem? 

Also, as stated before - this issue goes away if we downgrade to version before 1.56.0

**Comment by martinzugec-ctx** [comment]
@mjbvz I am using the combination of VSC with GitHub Desktop, but  others are using git + VSC (Bitbucket).

To reproduce:
1. Create fork from this repo: https://github.com/citrix/en-us-tech-zone
2. Check out using GitHub Desktop
3. Click on "Open in Visual Studio Code" in GHD
4. Open one of the markdown files (ideally under /poc-guides/ with screenshots)
5. Press Ctrl + K, V to open preview

No images are displayed:
![image](https://user-images.githubusercontent.com/53792597/119056997-ab4cbd00-b999-11eb-8154-99777ce162c7.png)

**Comment by mjbvz** [comment]
Simple repo:

1. In a workspace:

```
readme.md
cat.gif
```

2. Use an absolute image path in the md file:

```

![](/cat.gif)
```

**Bug**
The image doesn't load. Relative paths work fine

**Comment by mjbvz** [comment]
Should be fixed by 79dea51e79ac4bb48311a332adcada9a397bc59c

Let me know if this isn't workin in the next Insiders build
